<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAcolitoHabitaciones</class>
 <widget class="QWidget" name="MyAcolitoHabitaciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="AcolitoCustomLineEdit" name="buscarLineEdit"/>
       </item>
       <item>
        <widget class="QTableView" name="vistaHabitaciones"/>
       </item>
       <item>
        <widget class="QGroupBox" name="visualizarGroupBox">
         <property name="title">
          <string>Opciones de visualización</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelLibre">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Está libre</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="siLibreCheckBox">
            <property name="text">
             <string>Sí</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="noLibreCheckBox">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelLlaveEntregada">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Llave entregada</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="siEntregadaCheckBox">
            <property name="text">
             <string>Sí</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="noEntregadaCheckBox">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelOpcionesReserva">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reservadas</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="siReservadaCheckBox">
            <property name="text">
             <string>Sí</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="noReservadaCheckBox">
            <property name="text">
             <string>No</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="HabitacionWidget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#HabitacionWidget
{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.5, y2:0.46, stop:0 rgba(0, 227, 0, 255), stop:1 rgba(255, 255, 255, 255));
	border: 1px solid grey;
	border-radius: 5px;
}

QLineEdit#codHabLine
{
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
	
	color: rgb(0, 170, 0);
}

QComboBox#tipoComboBox
{	
    background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}


QTextBrowser#comentariosBrowser
{
	background-color: rgba(255, 255, 255, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QTextBrowser#reservaBrowser
{
	background-color: rgba(255, 255, 255, 0);
	color: rgb(255, 0, 0);
    border: no;
    
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="AcolitoBinaryToolButton" name="llaveButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Habitacion cerrada, pulsa para entregar la llave.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="acolitoRC.qrc">
             <normaloff>:/acolitos/iconos/habCerrada</normaloff>:/acolitos/iconos/habCerrada</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="AcolitoBinaryToolButton" name="reservaButton">
             <property name="toolTip">
              <string>Pulsa para reservar la habitación</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/noReserva</normaloff>:/acolitos/iconos/noReserva</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QToolButton" name="limpiarButton">
             <property name="toolTip">
              <string>Desaloja al grupo dejando la habitación vacía</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/limpiarHab</normaloff>:/acolitos/iconos/limpiarHab</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="imprimirButton">
             <property name="toolTip">
              <string>Imprimir habitacion</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/imprimir</normaloff>:/acolitos/iconos/imprimir</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QToolButton" name="reporteButton">
             <property name="toolTip">
              <string>Lanza un reporte de la habitación</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="acolitoRC.qrc">
               <normaloff>:/acolitos/iconos/exportar</normaloff>:/acolitos/iconos/exportar</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="habLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Tipo:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="AcolitoTextComboBox" name="tipoComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Comentarios:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextBrowser" name="comentariosBrowser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Sin comentarios&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="reservaLabel">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	color: rgb(255, 0, 0);
}</string>
             </property>
             <property name="text">
              <string>RESERVADA:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QTextBrowser" name="reservaBrowser">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Escriba el motivo de la reserva&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="codHabLine">
             <property name="text">
              <string>Código de habitación</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Alojados</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="vistaAlojados">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AcolitoTextComboBox</class>
   <extends>QComboBox</extends>
   <header>acolitotextcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AcolitoBinaryToolButton</class>
   <extends>QToolButton</extends>
   <header>acolitobinarytoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AcolitoCustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">acolitocustomlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="acolitoRC.qrc"/>
 </resources>
 <connections/>
</ui>
